app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/year_to_date_liability.scala.html

```diff
@@ -89,14 +89,14 @@
 
     @pageHeading(title)
 
-    @if(isATrust || representativeType.contains(PersonalRepresentativeInPeriodOfAdmin)){
-        <p>@Html(messages(s"$key$userKey.p1", viewConfig.trustsAndCgtUrl))</p>
+    @if(isATrust || representativeType.contains(PersonalRepresentativeInPeriodOfAdmin) || (representativeType.contains(PersonalRepresentativeInPeriodOfAdmin) && isAgent) ){
```

Comment:
the last `|| (representativeType.contains(PersonalRepresentativeInPeriodOfAdmin) && isAgent)` whck is redundant - the `|| representativeType.contains(PersonalRepresentativeInPeriodOfAdmin) ` will already capture the cases we want